apis/beacon/blobs/blobs.yaml

```diff
@@ -0,0 +1,53 @@
+get:
+  operationId: getBlobs
+  summary: Get blobs
+  description: |
+    Retrieves blobs for a given block id.
+    Depending on `Accept` header it can be returned either as json or as bytes serialized by SSZ.
+    It is recommended that blob retrieval for finalized blocks should by done by slot else the underlying index might not be available.
+  tags:
+    - Beacon
+  parameters:
```

Comment:
> For Prysm, we currently only index blobs by root and serve the BlobSidecarsByRange p2p request by finding the canonical chain of blocks for the given slot range and then looking up each set of blobs for a given slot by the canonical block's root. For us, requesting by slot would add some complexity to our API endpoint rather than making things simpler. That said, it's something we could support if other clients need it.

I _think_  Lighthouse is doing something similar to what you've described, I'm not sure if it's applicable either 🤔 if this is implementation-level detail, I'm inclined to leave it out of the spec - keen to hear thoughts!

Reply:
cc @realbigsean

Reply:
Added `indices` query parameter, thanks!

Reply:
Added in https://github.com/ethereum/beacon-APIs/pull/286/commits/27ac4b8e5486ab00efbef142b1ba3e024151d688